.env.sample

```
@@ -217,4 +217,25 @@ PROTOCOL=http
+# SENDGRID_API_KEY=SG.xxxxxxx
+
+# AWS SES (Fallback option)
+# AWS_SES_REGION=ap-south-1
+# AWS_SES_ACCESS_KEY=xxxx
+# AWS_SES_SECRET_KEY=xxxx
+
+# SMTP (Fallback option)
+# SMTP_HOST=smtp.gmail.com
+# SMTP_PORT=587
+# SMTP_USER=your-email@gmail.com
+# SMTP_PASS=xxxxxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)